android/app/build.gradle

```diff
@@ -136,7 +136,7 @@ android {
             signingConfig signingConfigs.debug
         }
         release {
-            signingConfig signingConfigs.debug
+            signingConfig signingConfigs.release
             productFlavors.production.signingConfig signingConfigs.release
```

Comment:
This line was also added and I don't think it does anything. But I don't want to remove it yet just in case

Reply:
I'm about 80% sure it can be deleted, but didn't want to take the risk. If this PR succeeds I'll follow up with a PR to remove it.